crates/next-custom-transforms/src/transforms/server_actions.rs

```diff
@@ -489,23 +784,55 @@ impl<C: Comments> VisitMut for ServerActions<C> {
             take(&mut self.names)
         };
 
-        if !is_action_fn {
-            return;
-        }
-
-        if !f.function.is_async {
+        if (is_action_fn || is_cache_fn) && !f.function.is_async {
             HANDLER.with(|handler| {
                 handler
                     .struct_span_err(f.function.span, "Server actions must be async functions")
                     .emit();
             });
         }
 
-        if !(self.in_action_file && self.in_export_decl) {
+        if is_cache_fn && !(self.in_cache_file && self.in_export_decl) {
+            // It's an action function. If it doesn't have a name, give it one.
```

Comment:
is this comment outdated? i thought this is a cache function, not an action function?